runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/util/BoundedQueueExecutorTest.java

```diff
@@ -39,13 +47,31 @@
 // released (2.11.0)
 @SuppressWarnings("unused")
 public class BoundedQueueExecutorTest {
-  @Rule public transient Timeout globalTimeout = Timeout.seconds(300);
   private static final long MAXIMUM_BYTES_OUTSTANDING = 10000000;
   private static final int DEFAULT_MAX_THREADS = 2;
   private static final int DEFAULT_THREAD_EXPIRATION_SEC = 60;
-
+  @Rule public transient Timeout globalTimeout = Timeout.seconds(300);
   private BoundedQueueExecutor executor;
 
+  private static ExecutableWork createWork(Consumer<Work> executeWorkFn) {
+    return ExecutableWork.create(
+        Work.create(
+            Windmill.WorkItem.newBuilder()
+                .setKey(ByteString.EMPTY)
+                .setShardingKey(1)
+                .setWorkToken(1)
```

Comment:
set this to some larger more unique token, since 1 could be in threadname anyway

Reply:
done also realized we format the work token so exposed that functionality to tests